packages/@react-oauth/google/src/GoogleLogin.tsx

```diff
@@ -13,6 +13,8 @@ import {
 const containerHeightMap = { large: 40, medium: 32, small: 20 };
 
 export type GoogleLoginProps = {
+  id: string;
+  className: string;
```

Comment:
These two should be optional I think

Reply:
yep true, i forgot to push that. making it required makes it a breaking change